cmd/agent-js-worker/src/agent.js

```diff
@@ -1055,14 +1055,31 @@ const Agent = function(opts) {
             pkgname: "didclient",
 
             /**
-             * Creates a DID.
+             * Creates a trustbloc DID.
              *
              * @param req - json document
              * @returns {Promise<Object>}
              */
-            createDID: async function (req) {
-                return invoke(aw, pending, this.pkgname, "CreateDID", req, "timeout while creating did")
+            createBlocDID: async function (req) {
```

Comment:
For consistency, should be `createTrustblocDID` if the pattern is "create" + did method + "DID"